### ISSUE TITLE
Low color contrast for table headers in light theme

### ISSUE THREAD

**Issue by biggeleben** [grafana/grafana#12668]
When using the light theme, readability of some elements is bad, e.g.
- .table-panel-table-header-inner (color: #61c2f2)
- .piechart-legend-table th (color: #61c2f2)

Maybe the color is used at other comparable situations as well.

Color contrast is just 2.00:1. Since the background is bright, the color needs to be darker to have a good contrast and readability (see e.g. https://www.w3.org/TR/WCAG20-TECHS/G18.html).

Colors that would work:
- #468fb3 (similar color, darker, contrast is 3.00:1)
- #3f7d9d (similar color, darker, contrast is 4.53:1, actually WCAG AA compliant)
- #757575 (grey, WCAG AA)
- #3e3e3e (almost black, even WCAG AAA)

I'm using Grafana v5.2.1 (2040f61). Would be awesome If someone spends a few minutes on picking a better color.

**Comment by marefr** [comment]
@biggeleben for the pie chart legend, please open an issue in the [piechart repository](https://github.com/grafana/piechart-panel)
